sequence_processing_pipeline/SIF.py

```diff
@@ -0,0 +1,12 @@
+
+
+class SampleInformationFile:
+    full_header = ['sample_name', 'collection_timestamp', 'elevation',
```

Comment:
We don't need this, right?

Reply:
We don't. We can go the other way where Pipeline contains this information until the time we really do need an abstraction.